docs/developer-llm-operator/install.md

```diff
@@ -0,0 +1,312 @@
+  or refer to the [meta-llama/LLama-2-13b-chat-hf](https://huggingface.co/meta-llama/Llama-2-13b-chat-hf)
+  page from HuggingFace.
+
+  The directory with the model is shared as a host path volume mount with the Triton Inference Server pod.
+
+
+## Install the NVIDIA GPU Operator
+
+Use the NVIDIA GPU Operator to install, configure, and manage the NVIDIA GPU driver and
+NVIDIA container runtime on the Kubernetes node.
+
+1. Add the NVIDIA Helm repository:
+
+   ```console
+   $ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
+    && helm repo update
+   ```
+
+1. Install the Operator:
+
+   ```console
+   $ helm install --wait --generate-name \
+    -n gpu-operator --create-namespace \
+    nvidia/gpu-operator
+   ```
+
+1. Optional: Configure GPU time-slicing if you have fewer than four GPUs.
+
+   - Create a file, `time-slicing-config-all.yaml`, with the following content:
+
+     ```yaml
+     apiVersion: v1
+     kind: ConfigMap
+     metadata:
+       name: time-slicing-config-all
+     data:
+       any: |-
+         version: v1
+         flags:
+           migStrategy: none
+         sharing:
+           timeSlicing:
+             resources:
+             - name: nvidia.com/gpu
+               replicas: 4
+     ```
+
+     The sample configuration creates four *replicas* from each GPU on the node.
+
+   - Add the config map to the Operator namespace:
+
+     ```console
+     $ kubectl create -n gpu-operator -f time-slicing-config-all.yaml
+     ```
+
+   - Configure the device plugin with the config map and set the default time-slicing configuration:
+
+     ```console
+     $ kubectl patch clusterpolicy/cluster-policy \
+         -n gpu-operator --type merge \
+         -p '{"spec": {"devicePlugin": {"config": {"name": "time-slicing-config-all", "default": "any"}}}}'
+     ```
+
+   - Verify that at least `4` GPUs are allocatable:
+
+     ```console
+     $ kubectl get nodes -l nvidia.com/gpu.present -o json |   jq '.items[0].status.allocatable |
+         with_entries(select(.key | startswith("nvidia.com/"))) |
+         with_entries(select(.value != "0"))'
+     ```
+
+     *Example Output*
+
+     ```output
+     {
+       "nvidia.com/gpu": "4"
+     }
+     ```
+
+For more information or to adjust the configuration, refer to
+[Install NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#install-nvidia-gpu-operator) and
+[Time-Slicing GPUs in Kubernetes](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-sharing.html)
+in the NVIDIA GPU Operator documentation.
+
+## Install the Developer LLM Operator
+
+1. Get the Helm chart for the Operator:
+
+   ```console
+   $ helm fetch https://helm.ngc.nvidia.com/nvidia/cloud-native/charts/developer-llm-operator-v0.1.0.tgz
+   ```
+
+1. Install the Operator:
+
+   ```console
+   $ helm install --generate-name ./developer-llm-operator-v0.1.0.tgz \
+       -n kube-trailblazer-system --create-namespace
```

Comment:
I believe that we need to remove the `v` from the semantic version part of the chart.

Reply:
Removed now!